Formula/dprint.rb

```diff
@@ -0,0 +1,50 @@
+class Dprint < Formula
+  desc "Pluggable and configurable code formatting platform written in Rust"
+  homepage "https://dprint.dev/"
+  url "https://github.com/dprint/dprint/archive/0.11.0.tar.gz"
+  sha256 "beb0e57cf3db632a2f2704b7e0cfca49be5c2d4428d19fa10b62b9dfd6d4d210"
+  license "MIT"
+  head "https://github.com/dprint/dprint.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "./crates/dprint"
```

Comment:
Isn't this `*std_cargo_args`?

Reply:
Close, but not exactly:
```
def std_cargo_args
  ["--locked", "--root", prefix, "--path", "."]
end
```

Reply:
Are we able to override the --path like we would for Go? I don't know anything about Cargo

Reply:
Neither do I. If I were forced to do it, I'd do something like
```
args = std_cargo_args.reject { |s| s["."] } + ["./crates/dprint"]
system "cargo", "install", *args
```
but this is prone to breaking if someone decides to modify `std_cargo_args`.

Reply:
If we can't override it I'm fine with the current version. It's worth a try though

Reply:
Actually, better:
```suggestion
    args = std_cargo_args.map { |s| s == "." ? "./crates/dprint" : s }
    system "cargo", "install", *args
```

Reply:
I think I tried before, it did not work. Let me give another try.